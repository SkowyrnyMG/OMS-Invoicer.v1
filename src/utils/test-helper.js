import React from 'react';
import { Router } from 'react-router-dom';
import { render, fireEvent } from '@testing-library/react';
import { ThemeProvider } from 'styled-components';
import { createMemoryHistory } from 'history';
import { theme } from 'themes/theme';

export const TestThemeProvider = ({ children }) => (
  <ThemeProvider theme={theme}>{children}</ThemeProvider>
);

const renderWithRouter = (ui, { route = '/', ...renderOptions } = {}) => {
  const history = createMemoryHistory({ initialEntries: [route] });
  const utils = render(<Router history={history}>{ui}</Router>, renderOptions);
  return {
    ...utils,
    history,
  };
};

const leftClick = { button: 0 };

export { leftClick, renderWithRouter, fireEvent };
