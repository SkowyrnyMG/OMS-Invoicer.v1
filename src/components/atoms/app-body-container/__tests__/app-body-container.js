import React from 'react';
import { snapShotTest } from 'utils/tests/test-helper';

import AppBodyContainer from '../app-body-container';

describe('AppBodyContainer', () => {
  test('Should display in document with correct styles', () => {
    snapShotTest(<AppBodyContainer />);
  });
});
