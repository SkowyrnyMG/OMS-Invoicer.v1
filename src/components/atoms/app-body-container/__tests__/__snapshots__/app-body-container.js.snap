// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AppBodyContainer Should display in document with correct styles 1`] = `
.c0 {
  height: 60vh;
  width: 100%;
  background-color: #FFFFFF;
  border-radius: 0.5rem;
  box-shadow: 0 0 10px -4px rgba(0,0,0,0.25);
  overflow: scroll;
}

@media (max-width:767px) {
  .c0 {
    height: 60vh;
  }
}

<div
  class="c0"
/>
`;
